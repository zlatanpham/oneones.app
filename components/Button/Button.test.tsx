import React from 'react'
import { render } from '@testing-library/react'
import { Button } from '.'

describe('<Button/>', () => {
  it('Should match snapshot', () => {
    const { container } = render(<Button>Click me</Button>)

    expect(container.firstChild).toMatchSnapshot()
  })
})
