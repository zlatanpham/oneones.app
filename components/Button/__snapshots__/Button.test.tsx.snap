// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button/> Should match snapshot 1`] = `
.c0 {
  padding-left: 1.25rem;
  padding-right: 1.25rem;
  padding-top: 0.75rem;
  padding-bottom: 0.75rem;
  background-color: #dd6b20;
  color: #fff;
  border-radius: 0.25rem;
}

<button
  class="c0"
>
  Hello
</button>
`;
